src/lib/unlang/xlat_builtin.c

```diff
@@ -1808,27 +1814,25 @@ static xlat_action_t xlat_func_bin(TALLOC_CTX *ctx, fr_dcursor_t *out,
 				   fr_value_box_list_t *in)
 {
 	fr_value_box_t		*result;
-	char			*buff = NULL, *p, *end;
+	char const		*p, *end;
 	uint8_t			*bin;
 	size_t			len, outlen;
 	fr_sbuff_parse_error_t	err;
+	fr_value_box_t		*in_head;
 
 	/*
 	 *	If there's no input, there's no output
 	 */
 	if (fr_dlist_empty(in)) return XLAT_ACTION_DONE;
```

Comment:
Remove this boilerplate if the arg is required

Reply:
I'm not sure `%{bin:}` is particularly useful?

Reply:
probably not...